Sources/StreamChat/WebSocketClient/WebSocketPingController.swift

```diff
@@ -6,15 +6,15 @@ import Foundation
 
 /// A delegate to control `WebSocketClient` connection by `WebSocketPingController`.
 protocol WebSocketPingControllerDelegate: AnyObject {
-    /// `WebSocketPingController` will call it method periodically to keep a connection alive.
+    /// `WebSocketPingController` will call this func periodically to keep a connection alive.
     func sendPing()
     
-    /// `WebSocketPingController` will call it to force disconnect `WebSocketClient`.
+    /// `WebSocketPingController` will call this func to force disconnect `WebSocketClient`.
```

Comment:
I would vote for either function or method, but not func

Reply:
Yeah sure ;)

Reply:
Done!